csharp/src/Microsoft.ML.OnnxRuntime/InferenceSession.cs

```diff
@@ -182,17 +201,30 @@ internal ModelMetadata ModelMetadata
 
         #region private methods
 
-        protected void Init(string modelPath, SessionOptions options)
+        /// <summary>
+        /// Initializes the session object with a model specified by either file path or byte array
+        /// </summary>
+        /// <param name="modelPath">Model's file path to load. This parameter is ignored if <paramref name="modelData"/> is specified.</param>
+        /// <param name="modelData">Model's byte array to read. If this parameter is ignored, then <paramref name="modelPath"/> will be ignored.</param>
+        /// <param name="options">Session options</param>
+        private void Init(string modelPath, byte[] modelData, SessionOptions options)
```

Comment:
Can this method be overloaded? Only one of modelPath and modelData can be present.

Reply:
Added two overloads accordingly.